### PR TITLE
doc: add detail of chaosctl debug

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What is changed and how does it work?
Add some explanation of what did `chaoctl debug` do

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/chaosctl/README.md [hunk 1]
```diff
@@ -6,28 +6,57 @@ Chaostl is a tool (currently) used to print debug info. Maintainers would make u
 ```shell
 cd $CHAOSMESH_DIR
 make chaosctl
-./bin/chaosctl --help #to see if build succeed.
+./bin/chaosctl --help # to see if build succeed.
```
Comment (fewdan) [inline pkg/chaosctl/README.md:9]: Seems to be changed to "succeeds"?

Comment (Yiyiyimu) [reply pkg/chaosctl/README.md]: Thanks! Fixed

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Comment by zhouqiang-cl** [comment]
@milasuperstar PTAL

**Comment by milasuperstar** [comment]
> @milasuperstar PTAL

LGTM

**Comment by Yiyiyimu** [comment]
@YangKeao @STRRL PTAL

**Review by fewdan — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@Yiyiyimu Can we move this doc to `cmd/chaosctl` directory and add a document on chaos mesh website?

**Pull request merged**